CSharp/Library/Microsoft.Bot.Connector.Shared/ConnectorClientEx.cs

```diff
@@ -49,25 +49,44 @@ public ConnectorClient(Uri baseUri, MicrosoftAppCredentials credentials, HttpCli
         {
             this.Credentials = credentials;
         }
-        
+
         private static DelegatingHandler[] AddJwtTokenRefresher(DelegatingHandler[] srcHandlers, MicrosoftAppCredentials credentials)
         {
             var handlers = new List<DelegatingHandler>(srcHandlers);
             handlers.Add(new JwtTokenRefresher(credentials));
             return handlers.ToArray();
         }
-        
-        // client defaults to sending the expect: continue header, which isn't very efficient, 
+
+        private HttpClient instanceClient;
+        protected static HttpClient g_httpClient = null;
+        protected static object syncObj = new object();

```

Comment:
g_syncObj ?

Reply:
Could we use one naming standard? :) 

_instanceClient
_httpClient
_syncRoot